site/content/en/docs/Getting Started/edit-first-gameserver-go.md

```diff
@@ -86,7 +79,7 @@ Apply the latest settings to the Kubernetes container.
 ```bash
 gcloud config set container/cluster [CLUSTER_NAME]
 gcloud container clusters get-credentials [CLUSTER_NAME]
-kubectl apply -f gameserver.yaml
+kubectl create -f gameserver.yaml
```

Comment:
I think apply should work - why change to create?

Reply:
I always ran into the following issue `error: from simple-game-server-: cannot use generate name with apply` and turns out that kubectl apply doesn't support `generateName`. Found some old random issues about it, https://github.com/tektoncd/pipeline/issues/2170,  https://github.com/kubernetes/kubernetes/issues/44501 and https://github.com/argoproj/argo-cd/issues/1639. The workaround is to use `kubectl create -f`.